config/_shared.json

```diff
@@ -144,5 +163,11 @@
 		{ "value": 452, "langSlug": "zh-tw", "name": "zh-tw - 繁體中文", "wpLocale": "zh_TW", "popular": 14 }
 	],
 	"project": "wordpress-com",
-	"twemoji_cdn_url": "https://s0.wp.com/wp-content/mu-plugins/wpcom-smileys/twemoji/2/"
+	"rtl": null,
+	"siftscience_key": null,
+	"signup_url": null,
+	"twemoji_cdn_url": "https://s0.wp.com/wp-content/mu-plugins/wpcom-smileys/twemoji/2/",
+	"wpcom_signup_id": null,
+	"wpcom_signup_key": null,
+	"wpcom_user_bootstrap": false
```

Comment:
All of these were added to satisfy the linting. Interestingly enough, only `desktop.json` had keys that were missing elsewhere.